test/Microsoft.AspNetCore.Mvc.Razor.Test/RazorPageTest.cs

```diff
@@ -296,7 +297,8 @@ public static TheoryData<HtmlEncoder> HtmlEncoderData
 
             // Act & Assert
             var ex = await Assert.ThrowsAsync<InvalidOperationException>(() => page.ExecuteAsync());
-            Assert.Equal("Section 'bar' is not defined in path '/Views/TestPath/Test.cshtml'.", ex.Message);
+            var message = $"The layout page '{context.ExecutingFilePath}' cannot find the section 'bar' in the content page '{context.View.Path}'.";
```

Comment:
Hmm why is this using string interpolation? This will essentially check that "the code does what the code does." By using a constant string you can verify that "the code produces this exact result."